pandas/tests/generic/test_frame.py

```diff
@@ -269,3 +270,40 @@ def test_deepcopy_empty(self):
         empty_frame_copy = deepcopy(empty_frame)
 
         self._compare(empty_frame_copy, empty_frame)
+
+    @pytest.mark.skipif(os.name == 'nt',
```

Comment:
if you use `tm.convert_rows_list_to_csv_str` this should become unnecessary.

Reply:
oh, yeah! nice!